### PR TITLE
fix(tests): fix failing tests

### LINKED ISSUE #7: Fix bug in `npm run test` result

### PR DESCRIPTION
**Pull request by MateusDadalto**
fixes #7

### CODE REVIEW

File: lib/finance.js [hunk 1]
```diff
@@ -174,7 +174,7 @@ var Finance = function (faker) {
     var address = faker.random.arrayElement(['1', '3']);
 
     for (var i = 0; i < addressLength - 1; i++)
-      address += faker.random.arrayElement('123456789abcdefghijkmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ'.split(''));
+      address += faker.random.arrayElement('123456789abcdefghijkmnopqrstuvwxyzABCDEFGHJKLMNPQRSTUVWXYZ'.split(''));
```
Comment (MateusDadalto) [inline lib/finance.js:177]: From what I've researched the test's validation regex is correct. The error on this one was with the generation.

The regex contains this matcher `[1-9a-km-zA-HJ-NP-Z]` which excludes lowercase l, uppercase I, and uppercase O. The generation method had uppercase I and uppercase O 

The same applies to litecoinAdress in this file

File: test/random.unit.js [hunk 2]
```diff
@@ -34,7 +34,7 @@ describe("random.js", function () {
     it('should return deterministic results when seeded with array - one element', function() {
       faker.seed([10]);
       var name = faker.name.findName();
-      assert.strictEqual(name, 'Duane Kub');
+      assert.strictEqual(name, 'Duane Kshlerin');
```
Comment (MateusDadalto) [inline test/random.unit.js:37]: I've checked multiple times and the seeded value is generatig Duane Kshlerin

**Comment by damienwebdev** [comment]
I'm holding off on merging this until #1 is merged.

**Review by Shinigami92 — APPROVED** [review]

**Comment by Shinigami92** [comment]
@damienwebdev IMO we can merge this now, so the main branch will be green
We could try to have an every-green main branch from then on :slightly_smiling_face:

**Review by damienwebdev — APPROVED** [review]
lgtm

**Pull request merged**